prototypes/python/otel.py

```diff
@@ -0,0 +1,127 @@
+
+import logging
+from typing import List, Sequence, Tuple
+from pkg_resources import iter_entry_points
+
+
+import yaml
+from pathlib import Path
+from jsonschema import validate, validators
+from jsonschema.exceptions import ValidationError
+
+from opentelemetry.metrics import set_meter_provider
+from opentelemetry.trace import set_tracer_provider
+from opentelemetry.sdk.trace.export import SpanExporter
+from opentelemetry.sdk.metrics.export import MetricExporter
+
+
+# borromed from opentelemetry/sdk/_configuration
+def _import_config_component(
+    selected_component: str, entry_point_name: str
+) -> object:
+    component_entry_points = {
+        ep.name: ep for ep in iter_entry_points(entry_point_name)
+    }
+    entry_point = component_entry_points.get(selected_component, None)
+    if not entry_point:
+        raise RuntimeError(
+            f"Requested component '{selected_component}' not found in entry points for '{entry_point_name}'"
+        )
+
+    return entry_point.load()
+
+class Config:
+    def __init__(self, config=None) -> None:
+        self._config = config
+    
+    def _resource(self):
+        # resource detection
+        # attributes
+        from opentelemetry.sdk.resources import Resource
```

Comment:
Why import here?

Reply:
not necessary to have the import inline, that was just me being lazy